packages/open-ui/src/components/Tag/Tag.module.scss

```diff
@@ -0,0 +1,16 @@
+@import 'styles/index.scss';
+
+.Tag {
+  background-color: $grey;
+  border-radius: 12px;
+  color: $black;
+  display: inline-block;
+  font-size: $font-size-sm;
+  padding: $vr;
```

Comment:
I'd like to see a little more horizontal padding I think maybe `2 *` on the the Y-axis